### PR TITLE
add __debug_bin file to gitignore

### PR DESCRIPTION
**Pull request by ghost**
this file being created while debugging with vscode on my side

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,6 +12,7 @@ _test
 
 # MS VSCode
 .vscode
+__debug_bin
```
Comment (jolheiser) [inline .gitignore:15]: Isn't it `__debug.bin`?
```suggestion
__debug.bin
```

**Review by jolheiser — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (jaqra) [reply .gitignore:15]: No. You can check [here](https://github.com/go-gitea/gitea/pull/8739/commits/20d6702bcd9490f1fc6d1240abc442338d396f20)

Comment (jolheiser) [reply .gitignore:15]: Ah, okay. I misread the first time. 🙃

**Review by jolheiser — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (jaqra) [reply .gitignore:15]: :)

Comment (jaqra) [reply .gitignore:15]: @jolheiser I also committed previous pr as `__debug.bin` (as message) :laughing:

**Review by lafriks — APPROVED** [review]

**Pull request merged**